keyboards/rastersoft/minitkl/minitkl.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2023 Raster Software Vigo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "minitkl.h"
+
+// It is possible to choose between the default driver and this custom one
+#ifdef RGBLIGHT_CUSTOM_DRIVER
+#include "rgblight.h"
+#include "config.h"
+
+LED_TYPE led[RGBLED_NUM];
+uint8_t *ledp = (uint8_t*)led;
+
+void rgblight_set() {
```

Comment:
This removes the need for the custom driver part:

```suggestion
void rgblight_call_driver(LED_TYPE *start_led, uint8_t num_leds) {
```

You'd need to change ledp to start_led in the rest of the function, but this also doesn't destroy the led map, layers, and clipping config.

Reply:
Done.